netboot-oci-artifacts.md

```diff
@@ -0,0 +1,96 @@
+# Packaging Network Boot files
+
+Netboot files (i.e. PXE) can be packaged as [OCI Artifacts](https://github.com/opencontainers/image-spec/blob/main/artifacts-guidance.md)
+using an [OCI image manifest](https://github.com/opencontainers/image-spec/blob/master/manifest.md)
+and stored in OCI registry.
+
+## Image config object
+
+The configuration object must have the MIME type `application/vnd.oci.empty.v1+json`, and must be
+an empty JSON object ('{}').
+See the [Guidance for an empty descriptor](https://github.com/opencontainers/image-spec/blob/main/manifest.md#guidance-for-an-empty-descriptor) 
+
+Because the `config.mediaType` is set to the empty value, the `artifactType` MUST be defined and
+currently it has MIME type `application/vnd.unknown.artifact.v1`.
+
+## Image layers
+
+Each layer in the manifest represents provided at build time PXE files:
```

Comment:
is there anything else to mention?

Reply:
This statement does not clearly say if each layer should contain just one file, or one layer can contain multiple files.

Also, please avoid PXE term, network booting is a generic concept and PXE is just one subset of that (intel x86_64 network booting). Tho I think it is okay to mention intel as an example:

For example, to boot Red Hat compatible system via PXE, the following files are subject of storing into a layer(s):

* shim.efi
* grubx64.efi
* vmlinuz
* initrd.img
* install.img

Reply:
Right, I initially used PXE everywhere and then changed to more generalized network boot, this is just a left over, I will remove it, thans for noticing!